tox.ini

```diff
@@ -51,7 +51,8 @@ passenv =
   AWS_DEFAULT_REGION
 deps =
   dnspython
-  pytest
+  pytest==3.3.2
+  pytest-catchlog==1.2.2
```

Comment:
I believe that this is not necessary:

See "As of pytest 3.3, the functionality of this plugin has been merged to the pytest core". on https://github.com/eisensheng/pytest-catchlog.